docs/usage.md

```diff
@@ -23,21 +23,26 @@ The second workflow uses [BWA-Meth](https://github.com/brentp/bwa-meth) and [Met
 
 ## Samplesheet input
 
-You will need to create a samplesheet with information about the samples you would like to analyse before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row as shown in the examples below.
+You will need to create a samplesheet with information about the samples you would like to analyse before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 4 columns, and a header row as shown in the examples below.
 
 ```bash
 --input '[path to samplesheet file]'
 ```
 
+```csv title="header.csv"
+sample,fastq_1,fastq_2,genome
+```
+
 ### Multiple runs of the same sample
 
 The `sample` identifiers have to be the same when you have re-sequenced the same sample more than once e.g. to increase sequencing depth. The pipeline will concatenate the raw reads before performing any downstream analysis. Below is an example for the same sample sequenced across 3 lanes:
 
 ```csv title="samplesheet.csv"
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L003_R1_001.fastq.gz,AEG588A1_S1_L003_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L004_R1_001.fastq.gz,AEG588A1_S1_L004_R2_001.fastq.gz
+sample,fastq_1,fastq_2,genome
+SRR389222_sub1,SRR389222_sub1.fastq.gz,,
+SRR389222_sub2,SRR389222_sub2.fastq.gz,,
+SRR389222_sub3,SRR389222_sub3.fastq.gz,,
+Ecoli,Ecoli_10K_methylated_R1.fastq.gz,Ecoli_10K_methylated_R2.fastq.gz,
```

Comment:
is `Ecoli` the sample or the genome?

Reply:
shouldn't `Ecoli_10K_methylated` be the sample?

Reply:
I was trying to just use the one in the test samplesheet. 

I can change it in the samplesheet but, will need to update all tests and snapshots

Reply:
insert interstellar meme - this one small step is going to cost us 7 years!!!! 🚀

Reply:
on it

Reply:
should be fixed now @maxulysse

Reply:
yeah sorry about that, but for me `Ecoli` could be actually a good usage of this new genome field.